packages/marketplace-ui/src/components/lbc/mint/candy-machine.ts

```diff
@@ -0,0 +1,495 @@
+import * as anchor from "@project-serum/anchor";
+
+import {
+  MintLayout,
+  TOKEN_PROGRAM_ID,
+  Token,
+  ASSOCIATED_TOKEN_PROGRAM_ID,
+} from "@solana/spl-token";
+import {
+  PublicKey,
+  Signer,
+  SystemProgram,
+  SYSVAR_SLOT_HASHES_PUBKEY,
+  TransactionInstruction,
+} from "@solana/web3.js";
+import { IMintArgs } from "@strata-foundation/marketplace-ui";
```

Comment:
```suggestion
import { IMintArgs } from "../MintButton";
```
Think this is killing build